backend/provisioner/testdata/go/echo/echo.go

```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/TBD54566975/ftl/go-runtime/ftl"
 )
 
-var db = ftl.PostgresDatabase("echodb")
+var EchoDb = ftl.PostgresDatabaseHandle
```

Comment:
Should we enforce a naming convention here? I noticed some are lower case and some are camel in the tests.

Reply:
I guess it doesn't matter, as the naming convention is enforced in the schema, and we probably don't want to force them all to be public.

Reply:
yeah, they all get converted to lower camel during extraction (and eventually, when supported, provisioning). you might need to export the DB, for ex. if you have a subpackage that uses it within the same module. in which case the actual name and type alias name would misalign

kind of a weird quirk. we also may need to think about compatibility with existing DBs because this approach loses some flexibility